src/Evolution/Executables/GeneralizedHarmonic/GeneralizedHarmonicBase.hpp

```diff
@@ -138,13 +140,24 @@ class CProxy_GlobalCache;
 }  // namespace Parallel
 /// \endcond
 
-struct GeneralizedHarmonicDefaults {
-  static constexpr size_t volume_dim = 3;
+template <bool UseDampedHarmonicRollon, typename EvolutionMetavarsDerived>
+struct GeneralizedHarmonicTemplateBase;
+
+template <bool UseDampedHarmonicRollon,
```

Comment:
@kidder Could you please make this a runtime option in a follow-up PR, perhaps open an issue?